pkg/storage/runtime_test.go

```diff
@@ -574,6 +576,77 @@ var _ = t.Describe("Runtime", func() {
 				)
 			})
 
+			It("should succeed to create a container", func() {
+				// When
+				info, err = sut.CreateContainer(&types.SystemContext{},
+					"podName", "podID", "imagename",
+					"8a788232037eaf17794408ff3df6b922a1aedf9ef8de36afdae3ed0b0381907b",
+					"containerName", "containerID", "",
+					0, "mountLabel", &idtools.IDMappings{})
+			})
+
+			It("should succeed to create a pod sandbox", func() {
+				// When
+				info, err = sut.CreatePodSandbox(&types.SystemContext{},
+					"podName", "podID", "imagename",
+					"8a788232037eaf17794408ff3df6b922a1aedf9ef8de36afdae3ed0b0381907b",
+					"containerName", "metadataName",
+					"uid", "namespace", 0, &idtools.IDMappings{})
+
+			})
+
+			AfterEach(func() {
+				// Then
+				Expect(err).To(BeNil())
+				Expect(info).NotTo(BeNil())
+				Expect(info.ID).To(Equal("id"))
+				Expect(info.Dir).To(Equal("dir"))
+				Expect(info.RunDir).To(Equal("runDir"))
+			})
+		})
+
+		t.Describe("success invalid (camel-case) image name", func() {
+			var (
+				info storage.ContainerInfo
+				err  error
+			)
+
+			BeforeEach(func() {
+				// Given
+				gomock.InOrder(
+					imageServerMock.EXPECT().GetStore().Return(storeMock),
+					storeMock.EXPECT().Image(gomock.Any()).Return(&cs.Image{}, nil),
+					imageServerMock.EXPECT().GetStore().Return(storeMock),
+					storeMock.EXPECT().GraphOptions().Return([]string{}),
+					storeMock.EXPECT().GraphDriverName().Return(""),
+					storeMock.EXPECT().GraphRoot().Return(""),
+					storeMock.EXPECT().RunRoot().Return(""),
+					imageServerMock.EXPECT().GetStore().Return(storeMock),
+					storeMock.EXPECT().Image(gomock.Any()).Return(&cs.Image{}, nil),
+					storeMock.EXPECT().Image(gomock.Any()).Return(&cs.Image{}, nil),
```

Comment:
This corresponds to https://github.com/openSUSE/cri-o/blob/e758da45bd7365196e7148425a6405830ebbdfc9/pkg/storage/runtime.go#L219 , doesn’t it? The code should never get that far with a camel-case name, and should _not_ succeed AFAICS, because all of
- https://github.com/openSUSE/cri-o/blob/e758da45bd7365196e7148425a6405830ebbdfc9/pkg/storage/runtime.go#L167
- https://github.com/openSUSE/cri-o/blob/e758da45bd7365196e7148425a6405830ebbdfc9/pkg/storage/runtime.go#L170
- https://github.com/openSUSE/cri-o/blob/e758da45bd7365196e7148425a6405830ebbdfc9/pkg/storage/runtime.go#L176

fails with an error.

Reply:
Yeah I think the test case does not represent a real world-scenario... I think it would be acceptable to remove it completely, WDYT?

Reply:
I agree.

Reply:
Will be done in https://github.com/kubernetes-sigs/cri-o/pull/2132